custom-metrics-api/custom-metrics-apiserver-deployment.yaml

```diff
@@ -24,16 +24,27 @@ spec:
               memory: "128Mi"
             limits:
               memory: "128Mi"
-        image: quay.io/coreos/k8s-prometheus-adapter-amd64:v0.2.0
+        image: quay.io/coreos/k8s-prometheus-adapter-amd64:v0.4.1
         args:
-        - /adapter
         - --secure-port=6443
         - --logtostderr=true
         - --prometheus-url=http://prometheus.monitoring.svc:9090/
         - --metrics-relist-interval=30s
-        - --rate-interval=5m
-        - --v=10
+        - --v=2
```

Comment:
why are we going back to version 2? or is it verbosity?
and why are we removing the `rate-interval`?

Reply:
Yes, this is log verbosity. v=10 gives you all logs, with all communication done to the api server. V=2 still gives us enoying logs since we are not using certificates, but not nearly as much logs. 
rate-interval was removed after alpha version 0.2.0 as parameter